python/ray/train/callbacks/profile.py

```diff
@@ -2,76 +2,57 @@
 import pathlib
 from typing import List, Dict, Optional, Union
 
-from torch.profiler import profile, record_function, schedule
+from torch.profiler import profile
 
 from ray import train
 from ray.train import TrainingCallback
-from ray.train.callbacks.logging import TrainingLogdirMixin
+from ray.train.callbacks.logging import TrainingLogdirCallback
 from ray.train.callbacks.results_prepocessors import KeysResultsPreprocessor, \
     WorkersResultsPreprocessor
 from ray.train.constants import PROFILER_KEY
 
 logger = logging.getLogger(__name__)
 
-WORKER_TRACE_DIR = "worker_traces"
+WORKER_TRACE_DIR = "/tmp/pytorch_profiler_worker_traces"
```

Comment:
will this work on Windows? you probably want a `os.tmpdir` + `pathlib` combo here